test/cpp/jit/test_lite_interpreter.cpp

```diff
@@ -505,7 +505,7 @@ TEST(LiteInterpreterTest, SequentialModuleInfo) {
   }
 
   std::unordered_set<std::string> expected_result(
-      {"top(C).A0(A).forward", "top(C).B0(B).forward"});
+      {"top(C).A0(A).forward", "top(C).A0(A).B0(B).forward"});
```

Comment:
Why this test case was changed? Give B0(B) and A0(A) are inside C you should not see top(C).A0(A).B0(B).

Reply:
Module C forward methods is return self.A0.forward(self.B0.forward(x)), so the expected result should be top(C).A0(A).B0(B).forward, do I understand correctly?